setup.cfg

```
@@ -73,3 +74,6 @@ plotly =
     plotly==4.9.0
     psutil==5.7.0
     kaleido
+
+[options.package_data]
+* = tests/data/*
```

Following the example in nibabel: https://github.com/nipy/nibabel/blob/master/setup.cfg#L82
```suggestion
AFQ = tests/data/*
```